packages/vite/src/node/server/index.ts

```diff
@@ -204,6 +205,7 @@ export interface ResolvedServerOptions
   > {
   fs: Required<FileSystemServeOptions>
   middlewareMode: NonNullable<ServerOptions['middlewareMode']>
+  hostname: Hostname
```

Comment:
```suggestion
  /** @internal */
  hostname: Hostname
```
I'd like to mark this as a internal type.

Reply:
Makes sense, thanks for the feedback.